cmake/config/linux-gcc.txt

```diff
@@ -253,6 +253,14 @@ macro (link_against_seexpreditor target)
     target_link_libraries (${target} ${SEEXPREDITOR_LIBRARIES})
 endmacro ()
 
+macro (link_against_embree target)
+    if (USE_EXTERNAL_EMBREE)
+        target_link_libraries (${target} ${EMBREE_LIBRARIES})
+    else ()
+        #TODO
```

Comment:
Please do.

Reply:
I don't know what "external Embree" means in terms of linux build... Will be there once Windows build is there